runtime/compiler/aarch64/codegen/ARM64PrivateLinkage.hpp

```diff
@@ -106,6 +106,31 @@ class ARM64PrivateLinkage : public TR::Linkage
     */
    virtual void createEpilogue(TR::Instruction *cursor);
 
+   /**
+    * @brief Loads all parameters passed on the stack from the interpreter into
+    *        the corresponding JITed method private linkage register.
+    *
+    * @param[in] cursor : the TR::Instruction to begin generating
+    *        the load sequence at.
+    *
+    * @return : the instruction cursor after the load sequence
+    */
+   TR::Instruction *loadStackParametersToLinkageRegisters(TR::Instruction *cursor);
+
+   /**
+    *
+    * @brief Copies parameters from where they enter the method (either on stack or
+    *        in a linkage register) to their "home location" where the method body
+    *        will expect to find them (either on stack or in a global register).
+    *
+    * @param[in] cursor : the instruction cursor to begin inserting copy instructions
+    * @param[in] allParmsHaveBeenSpilled : all parameters are already present in stack
```

Comment:
This parameter's value seems dubious. Is it preferable to simply not call this function if this condition is true?

Reply:
Yes, I was thinking too far down the road.  I've just removed it for now.